_posts/platform/app/2000-01-01-log-drain.md

```diff
@@ -32,24 +33,101 @@ Our system is also able to send logs to the following SaaS log processing provid
 
 ## Configuration of your Application on Scalingo
 
-As this feature is private beta, you cannot activate logs drain by yourself. You need to add an
-environment variable containing the token to the used logs management service, and contact the
-support so that we activate the logs drain. In a matter of minutes, your logs flow to the logs
-management service.
+The log drains configuration will be done using the [CLI
+](https://doc.scalingo.com/cli).
+
+### Add a Log Drain
+
+To add a log drain management service you will have to use the `log-drains-add`
+command.
+Below, the list of available parameters:
```

Comment:
Why a new line here? Either you need a new paragraph and you need an extra new line, or you are in the same paragraph and the new line is not necessary.